packages/go_router/README.md

```diff
@@ -223,13 +46,10 @@ See [examples](https://github.com/flutter/packages/tree/main/packages/go_router/
 - [Migrating to 5.1.2](https://flutter.dev/go/go-router-v5-1-2-breaking-changes)
```

Comment:
Would it be better if these migration guides were ordered in reverse? In theory, fewer folks need to migrate from older versions as time goes on. I don't feel strongly about this, just a thought.